src/robot/libdocpkg/htmlwriter.py

```diff
@@ -129,7 +132,8 @@ def html(self, doc, intro=False):
     def _link_keywords(self, match):
         name = match.group(1)
         if name in self._targets:
-            return '<a href="#%s" class="name">%s</a>' % (self._targets[name], name)
+            return '<a href="#%s" class="name">%s</a>' % (
+            self._targets[name], name)
```

Comment:
This new formatting, more precisely the indentation of the second like, is very badly against PEP-8 and our conventions. In my opinion it would be best to leave it like it was, but if you want to change it you need to use one of these alternatives:

```python
return ('a template that is pretty long'
        % (some, values))

return 'a template that is pretty long' \
        % (some, values)

return 'a template that is pretty long' % (some,
                                           values)
```